test/pll_set_hz.c

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include <stdio.h>
+#include <mee/clock.h>
+
+#define PLL_MIN_OUT 16000000
+#define PLL_MAX_OUT 320000000
+
+void main() {
+    /* TODO: have DTS parser generate handle for PLL? */
+    struct mee_clock * pll = &(__mee_dt_clock_4.clock);
```

Comment:
We need a better interface here, as this will only work on the HiFive1 (and even then, only with the device tree exactly as it is currently specified).  Instead we should add functionality into the MEE for enumerating cores and then finding the clocks that drive them.

It's fine to leave this in as a test case for now, but there should be some way of ensuring it compiles for other targets.